.github/workflows/test.yml

```diff
@@ -171,6 +171,17 @@ jobs:
         name: runtests-${{ runner.os }}-py${{ matrix.python-version }}-Salt${{ matrix.salt-version }}.log
         path: artifacts/runtests-*.log
 
+    - name: Publish distribution 📦 to Test PyPI
+      if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        repository_url: https://test.pypi.org/legacy/
+        print-hash: true
+        skip-existing: true
+        verbose: true
+        verify-metadata: true
+
+
```

Comment:
```suggestion
```

Removing for suggested replacement in later suggestion (in order to have a dedicated `Build` step for publishing).

Reply:
okay will look at doing that tomorrow

Reply:
done